<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Enter_XML_File">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter_XML_File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decompress_Button">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>550</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Minify_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Minify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Correct_Errors_Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct_Errors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_xml">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>upload_xml</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Compress_Button">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>581</width>
      <height>471</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>579</width>
       <height>469</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Convert_To_Json">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>550</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert_To_Json</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Format_Button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_Json_Button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>550</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save_Json</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>581</width>
      <height>471</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>579</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuds_ali_l_gamed">
    <property name="title">
     <string>ds ali l gamed</string>
    </property>
   </widget>
   <addaction name="menuds_ali_l_gamed"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
